composer.json

```diff
@@ -18,7 +18,10 @@
     ],
     "require": {
         "yiisoft/yii2": "^2.0.6",
-        "devgroup/yii2-tag-dependency-helper": "^1.2.0"
+        "jisoft/yii2-sypexgeo": "dev-master",
+        "devgroup/yii2-tag-dependency-helper": "^1.2.0",
+        "devgroup/yii2-multilingual": "dev-master",
+        "devgroup/yii2-multilingual-sypexgeodaemon": "dev-master"
```

Comment:
Why this package needs sypex dependencies? Geo resolving can be achieved with other methods.